frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -1710,4 +1710,5 @@ ACTION_TYPE_FAILED_GLUSTER_STORAGE_CONNECTION_UNSUPPORTED_STORAGE_TYPE=Unsupport
 ACTION_TYPE_FAILED_CONVERT_DISK_NO_FORMAT_CHANGE=Cannot ${action} ${type}. Disk is already in format ${format}
 ACTION_TYPE_FAILED_CONVERT_DISK_NO_ALLOCATION_CHANGE=Cannot ${action} ${type}. Disk is already in allocation ${allocation}
 ACTION_TYPE_FAILED_DISK_WITH_CHAIN=Cannot ${action} ${type}. Disks with multiple snapshots are not supported.
-ACTION_TYPE_FAILED_HOSTED_ENGINE_VM_BACKUP_NOT_SUPPORTED=Cannot ${action} ${type}. Backup of Hosted Engine VM is not supported
\ No newline at end of file
+ACTION_TYPE_FAILED_HOSTED_ENGINE_VM_BACKUP_NOT_SUPPORTED=Cannot ${action} ${type}. Backup of Hosted Engine VM is not supported
+ACTION_TYPE_FAILED_ACTIVE_IMAGE_TRANSFER_FOR_VM_BACKUP=Cannot ${action} ${type}. There is an active image transfer for VM backup
```

Comment:
Isn't it worth to add backup ID? And a newline character at the end of the line?

Reply:
Doesn't codestyle start yelling at you if you put a newline there?